src/main/kotlin/pt/up/fe/ni/website/backend/model/seeders/RoleSeeder.kt

```diff
@@ -0,0 +1,43 @@
+package pt.up.fe.ni.website.backend.model.seeders
+
+import org.springframework.stereotype.Component
+import pt.up.fe.ni.website.backend.config.Logging
+import pt.up.fe.ni.website.backend.model.Role
+import pt.up.fe.ni.website.backend.model.permissions.Permission
+import pt.up.fe.ni.website.backend.model.permissions.Permissions
+import pt.up.fe.ni.website.backend.repository.RoleRepository
+
+@Component
+class RoleSeeder() :
+    AbstractSeeder<RoleRepository, Role, Long>(), Logging {
+
+    override fun createObjects() {
+        logger.info("Running Role seeder...")
+
+        if (repository.count() > 0) {
+            logger.info("Skipping Role seeding as roles already exist.")
+            return
+        }
+
+        val roles = listOf(
+            Role(
+                name = "President",
+                isSection = true,
+                permissions = Permissions(listOf(Permission.EDIT_ACTIVITY))
+            ),
+            Role(
+                name = "Vice President",
+                isSection = true,
+                permissions = Permissions(listOf(Permission.EDIT_ACTIVITY))
+            ),
+            Role(
+                name = "Member",
+                isSection = false,
+                permissions = Permissions(listOf(Permission.VIEW_ACTIVITY))
+            )
+        )
```

Comment:
I think there should be a couple more sections (`isSection = true`) to test the frontend, and maybe make them match the frontend for it to be less confusing! (Aka Direção Membros Recrutas Alumni)